cli/cmd/transformCmd.go

```diff
@@ -7,6 +7,7 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/jf-tech/go-corelib/ios"
```

Comment:
- What does ios stands for?

Reply:
All those packages named `<XYZ>s` (like `jsons`, `strs`, `times` etc) to avoid naming collision with stdlib packages. `ios` is from `io` but yeah I know it might be a bit funny , but for the sake of consistency I kept it that way.